### DISCUSSION TITLE
Is it possible to recuperate a non saved .pdfpc?

### DISCUSSION THREAD

**Discussion by aestebant** [pdfpc/pdfpc#734] [Q&A] [answered]
I would like to know if there is some possibility of recuperating a non saved file .pdfpc associated to a presentation from some temporal folder where it may be. I have installed pdfpc in MacOS through Homebriew, version 4.6. Thank you so much in advance.

**Comment by fnevgeny** [comment, marked as answer]
Sorry, I somehow entirely missed this topic. So no, there is no temporal folder or anything like that; when pdfpc exits, the .pdfpc file is saved in the same directory as the presentation.
